scripts/run-live-build.sh

```diff
@@ -61,6 +61,43 @@ else
 	cd "$TOP/live-build/variants/$1"
 fi
 
+function get_s3_uri() {
+
+	local def_bucket="snapshot-de-images"
+	local jenkinsid="jenkins-ops"
+
+	local prefix="$1"
+	local latest="s3://$def_bucket/builds/$jenkinsid/$2"
```

Comment:
if we're going to "bake in" parts of the URI like "builds" here, should we also do that with "post-push/latest"?

Reply:
We could do that.